src/Main.java

```diff
@@ -1,4 +1,67 @@
+import java.time.LocalDate;
+
 public class Main {
     public static void main(String[] args) {
+        System.out.println("Задание 1");
+        int year = 2025;
+        checkYear(year);
+
+        System.out.println("\nЗадание 2");
+        int clientOS = 1;
+        int clientDeviceYear = 2014;
+        checkOS(clientOS, clientDeviceYear);
+
+        System.out.println("\nЗадание 3");
+        int deliveryDistance = 95;
+        int days = calculateDays(deliveryDistance);
+        if (days > 0) {
+            System.out.print("Потребуеться дней на доставку: " + days);
+        } else {
+            System.out.println("Доставка на дакую дистанцию не возможна");
+        }
     }
+
+    public static void checkYear(int year) {
+        System.out.print(year + " год - ");
+        if (year % 100 == 0 && year % 400 != 0) {
+            System.out.print("не високосный");
+        } else if (year % 400 == 0) {
+            System.out.println("високосный");
+        } else if (year % 4 == 0) {
+            System.out.println("високосный");
+        } else {
+            System.out.println("не високосный");
+        }
```

Comment:
Здесь должен быть 1 if и 1 else, в прошлый раз писал про это. Условие високосного года одно, оно не состоит из частей, которые можно проверить по отдельности. Год либо високосный, либо нет, всё. Код должен был быть вот такой в итоге:
```suggestion
        if (year % 4 == 0 && year % 100 != 0 || year % 400 == 0) {
            System.out.print("високосный");
        } else {
            System.out.println("не високосный");
        }
```